crates/re_types/definitions/rerun/components/position3d.fbs

```diff
@@ -9,11 +9,11 @@ namespace rerun.components;
 
 // ---
 
-/// A point in 3D space.
-struct Point3D (
+/// A position in 3D space.
+struct Position3D (
   "attr.python.aliases": "npt.NDArray[np.float32], Sequence[float], Tuple[float, float, float]",
   "attr.python.array_aliases": "npt.NDArray[np.float32], Sequence[float]",
-  "attr.rerun.legacy_fqname": "rerun.point3d",
+  "attr.rerun.legacy_fqname": "rerun.position3d",
```

Comment:
Wrong!? I guess it doesn't matter a lot right now tough

Reply:
I wasn't quite sure about this one actually. We use new name with all other legacy names, so that's why I opted for position3d

Reply:
removed alltogether